miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/element/HandsOnTable.java

```diff
@@ -56,16 +58,32 @@ public int getRowCount() {
 
   public String getText(String columnHeading, int rowNum) {
     WebElement cell = getCell(columnHeading, rowNum);
-    return cleanOptionLabel(cell.getText());
+    return cleanAscii(cell.getText());
   }
 
+  /**
+   * Enters text into a table cell
+   * 
+   * @param columnHeading
+   * @param rowNum 0 is the first data row
+   * @param text text to enter. Only ASCII characters are accepted
+   * @throws IllegalArgumentException if text contains non-ASCII characters
+   */
   public void enterText(String columnHeading, int rowNum, String text) {
+    if (!text.equals(cleanAscii(text))) {
+      throw new IllegalArgumentException("text contains non-ASCII characters");
+    }
     WebElement cell = getCell(columnHeading, rowNum);
-    new Actions(getDriver())
+    Action action = new Actions(getDriver())
         .click(cell)
+        .sendKeys(Keys.DELETE)
         .sendKeys(text)
         .sendKeys(Keys.ENTER)
-        .build().perform();
+        .build();
+    // Odd behaviour of Chrome webdriver on Travis CI causes the first attempt to fail sometimes, so try up to 3 times
+    for (int retries = 0; retries < 3 && !text.equals(getText(columnHeading, rowNum)); retries++) {
+      action.perform();
```

Comment:
I did that at first, but then realized the method is also used in ways where it is expected to fail such as ensuring that fields cannot be written to. I think it's ideal to use the same method in testing both cases rather than have separate method that works differently when a failure is expected. That way, it's up to the test to assert the outcome as it should